src/test/java/com/iabtcf/decoder/TCStringV2Test.java

```diff
@@ -117,6 +118,14 @@ public void testCanParseAllParts() {
         assertEquals(1, IntIterableUtils.toStream(tcModel.getPubPurposesConsent()).count());
     }
 
+    @Test
+    public void testCanParseRangeEncodedVendorLegitimateInterests() {
+        String base64CoreString =
+                "COv__-wOv__-wC2AAAENAPCgAAAAAAAAAAAAA_wAQA_gEBABAEAAAA";
+        TCString tcModel = parse(base64CoreString);
+        assertEquals(new TreeSet<>(Arrays.asList(128)), IntIterableUtils.toSet(tcModel.getVendorLegitimateInterest()));
```

Comment:
you can use,

import static com.iabtcf.test.utils.IntIterableMatcher.matchInts;

assertThat(tcModel.getVendorLegitimateInterest(), matchInts(128));

Reply:
fixed